pkg/controller/configmap/configmap_controller.go

```diff
@@ -101,17 +108,37 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 		reqLogger.Info("Reconciling nsx-ncp-operator ConfigMap change")
 	} else if request.Namespace == "" && request.Name == operatortypes.NetworkCRDName {
 		reqLogger.Info("Reconciling cluster Network CRD change")
+	} else if request.Namespace == operatortypes.OperatorNamespace && request.Name == operatortypes.NcpInstallCRDName {
+		reqLogger.Info("Reconciling ncp-install CRD change")
 	} else {
 		return reconcile.Result{}, nil
 	}
 
+	// Fetch ncp-install CRD instance
+	ncpInstallCrd := &operatorv1.NcpInstall{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: operatortypes.NcpInstallCRDName,
+		Namespace: operatortypes.OperatorNamespace}, ncpInstallCrd)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			log.Info(fmt.Sprintf("%s CRD is not found", operatortypes.NcpInstallCRDName))
+			r.status.SetDegraded(statusmanager.OperatorConfig, "NoNcpInstallCRD",
+				fmt.Sprintf("%s CRD is not found", operatortypes.NcpInstallCRDName))
+			return reconcile.Result{}, nil
+		}
+		// Error reading the object - requeue the request.
+		r.status.SetDegraded(statusmanager.OperatorConfig, "NoNcpInstallCRD",
```

Comment:
do we need to unset `"NoNcpInstallCRD"`?

Reply:
It should be done in line 327